themes/V3/Blank/snippets.js

```diff
@@ -128,6 +129,44 @@ module.exports = [
 			}
 		]
 	},
+
+	{
+		groupName : 'License',
+		icon	  : 'fas fa-copyright',
+		view	  : 'text',
+		snippets  : [
+			{
+				name : 'CC BY 4.0',
+				icon : 'fab fa-creative-commons',
+				gen	 : LicenseGen.cc,
+			},
+
+			{
+				name : 'OGL 1.0 A',
+				icon : 'fab fa-wizards-of-the-coast',
+				gen	 : LicenseGen.OGL1,
+			},
+
+			{
+				name : 'WoTC Fan Content Policy',
+				icon : 'fas fa-w',
+				gen	 : LicenseGen.FCP,
+			},
+
+			{
+				name : 'ORC License',
+				icon : 'fas fa-Paizo',
```

Comment:
Looks like we need a working icon for this. I don't think the ORC license has its own logo either. We may need to just add a custom icon of the paizo logo.